app/lib/search/index_simple.dart

```diff
@@ -770,14 +796,16 @@ class TokenIndex {
 
   /// Returns an {id: score} map of the documents stored in the [TokenIndex].
   /// The tokens in [tokenMatch] will be used to calculate a weighted sum of scores.
+  /// When [ids] is specified, the result will contain only the set of ids in it.
```

Comment:
Wouldn't be better to name this `packages` or `packageIds` or `onlyPackageIds` or `limitToPackageId`? Since `ids` is really a list of package names, right? or is it something else?